### DISCUSSION TITLE
能支持快手吗

### DISCUSSION THREAD

**Discussion by loveqianool** [Tairraos/TikDown#18] [Ideas]
需要在多个渠道下载素材

**Comment by Tairraos** [comment]
太长不看版：我做不出来。

我看了下快手，视频本来就没有水印。
快手至少有两种视频，chrome/firefox/edge浏览器看视频在视频上右键，inspect，大概是20秒以内的，直接可以看到mp4地址，在地址上再次右键，open in a new tab，然后ctrl-s就能存下mp4来
有些几分钟的视频，地址显示成blob:https://xxxx 这样的，是媒体流，真正的源文件CDN地址被藏起来了，需要有能获取CDN文件地址的API才能下载。而我不知道快手的api。

**Reply by loveqianool** [reply]
https://github.com/5ime/video_spider
直接用他的 API 吗，或者可以自定义解析 API，GitHub 上挺多解析项目的，某些用户可能会自建，可能自定义 api 会好点

**Reply by Tairraos** [reply]
谢谢，你推荐的这个项目不错。
我并不需要使用它二次包装的API，可以从源文件里找到视频网站原生API的用法，直接使用。

我有空的时候会加上快手视频的，做好了我会让你知道。

**Reply by loveqianool** [reply]
thx
